website/src/content/deployment-examples/commercetools-connect.mdx

```diff
@@ -0,0 +1,119 @@
+3. Add your environment variables to `custom-application-config.mjs`.
+
+    ```js title="custom-application-config.mjs" highlightLines="5-13"
+    const config = {
+      name: 'Example Custom Application',
+      entryPointUriPath,
+      cloudIdentifier: '${env:CLOUD_IDENTIFIER}',
+      env: {
+        development: {
+          initialProjectKey: '${env:CTP_PROJECT_KEY}',
+        },
+        production: {
+          applicationId: '${env:CUSTOM_APPLICATION_ID}',
+          url: '${env:APPLICATION_URL}',
+        },
+      },
+      oAuthScopes: {
+        view: ['view_products'],
+        manage: ['manage_products'],
+      },
+      icon: '${path:@commercetools-frontend/assets/application-icons/rocket.svg}',
+      mainMenuLink: {
+        defaultLabel: 'Template starter',
+        labelAllLocales: [],
+        permissions: [PERMISSIONS.View],
+      },
+      submenuLinks: [
+        {
+          uriPath: 'channels',
+          defaultLabel: 'Channels',
+          labelAllLocales: [],
+          permissions: [PERMISSIONS.View],
+        },
+      ],
+    };
+
+    export default config;
+    ```
+
+4. Open `connect.yaml` in the root of the Connect starter template and replace it with the following. The environment variables you included in `custom-application-config.mjs` should be defined here. Sensitive variables, such as API client information, should be defined within `securedConfiguration`.
+
+      ```yml
+      deployAs:
+        - name: example-custom-application
+          applicationType: mc-app
+          endpoint: /
+          configuration:
+            standardConfiguration:
+              - key: CTP_PROJECT_KEY
+                description: project key from the Composable Commerce Project
+                default: "your-project-key"
+              - key: CTP_REGION
+                description: region of the Composable Commerce Project
+                required: true
+              - key: CUSTOM_APPLICATION_ID
+                description: the Custom Application ID
+                required: true
+              - key: CLOUD_IDENTIFIER
+                description: the cloud identifier
+                default: "gcp-eu"
+            securedConfiguration:
+              - key: CTP_CLIENT_ID
+                description: client id from the Composable Commerce Project
+                required: true
+              - key: CTP_CLIENT_SECRET
+                description: client secret from the Composable Commerce Project
+      ```
+
+<Info>
+
+`APPLICATION_URL` is automatically provided by commercetools Connect and you do not need to define it in `connect.yaml`.
+
+</Info>
+
+5. Push your code to a GitHub repository and make a new release. Your release must have a Git tag.
+
+<Info>
+
+Your GitHub repository can be public or private. If you are using a private repository, you must provide access to the GitHub machine user [connect-mu](https://github.com/connect-mu).
+
+</Info>
```

Comment:
Does this relate to "creating a git tag and making a new release"? Or does it refer to something else?

Reply:
It refers to the "push your code to a GitHub repository" part of the step. A private GH repository needs the machine user.